source/features/filter-comments-by-you.tsx

```diff
@@ -1,21 +1,18 @@
-import React from 'dom-chef';
 import select from 'select-dom';
 import features from '../libs/features';
-import {getRepoURL} from '../libs/utils';
+import {getUsername, getRepoURL} from '../libs/utils';
 
 const repoUrl = getRepoURL();
 
 function init(): void {
-	select('.subnav-search-context .SelectMenu-list a:last-child')!
-		.before(
-			<a
-				href={`/${repoUrl}/issues?q=is%3Aopen+commenter:@me`}
-				className="SelectMenu-item"
-				role="menuitem"
-			>
-				Everything commented by you
-			</a>
-		);
+	const dropDownElement = select('#filters-select-menu a:last-child, .subnav-search-context li:last-child')!;
+	const newDropDownElement = dropDownElement.cloneNode(true);
+	const href = `/${repoUrl}/issues?q=is%3Aopen+commenter:${getUsername()}`;
```

Comment:
It's already cached, no need for another variable

```suggestion
	const href = `/${getRepoURL()}/issues?q=is%3Aopen+commenter:${getUsername()}`;
```